cmd/ko/commands.go

```diff
@@ -295,4 +320,113 @@ func addKubeCommands(topLevel *cobra.Command) {
 	addTagsArg(run, ta)
 
 	topLevel.AddCommand(run)
+
+	// TODO: Write docs
+	py := &cobra.Command{
```

Comment:
`py` - I originally thought we were supporting python.

Call it `copy` please lol

Reply:
But the puns!

Reply:
What about `ko migrate`?

comigrate (verb): (chiefly in biochemistry) To migrate together